contrib/executor/jmeterd/pkg/slaves/utils.go

```diff
@@ -218,3 +131,22 @@ func getSlavesCount(count testkube.Variable) (int, error) {
 	}
 	return rplicaCount, err
 }
+
+func GetSlavesIpString(podNameIpMap map[string]string) string {
+	podIps := []string{}
+	for _, ip := range podNameIpMap {
+		podIps = append(podIps, ip)
+	}
+	return strings.Join(podIps, ",")
+}
+
+func ValidateAndGetSlavePodName(testName string, executionId string, currentSlaveCount int) string {
+	slavePodName := fmt.Sprintf("%s-slave-%v-%s", testName, currentSlaveCount, executionId)
+	if len(slavePodName) > 64 {
+		//Get first 20 chars from testName name if pod name > 64
+		shortExecutionName := testName[:20]
+		slavePodName = fmt.Sprintf("%s-slave-%v-%s", shortExecutionName, currentSlaveCount, executionId)
```

Comment:
how long is the execution ID?

Reply:
Hey  @sahil-lakhwani 
They generates executionId using mongo objectId func
if id == "" {
		id = primitive.NewObjectID().Hex()
	}
This primitive.NewObjectID() return byte[12] 
In MongoDB, ObjectID is a 12-byte identifier typically represented as a 24-character hexadecimal string
so that we can assume that executionId is of 24 chars

Reply:
ok, thanks for checking.